schemas/stsci.edu/asdf/0.1.0/core/history_entry.yaml

```diff
@@ -0,0 +1,28 @@
+%YAML 1.1
+---
+$schema: "http://stsci.edu/schemas/yaml-schema/draft-01"
+id: "http://stsci.edu/schemas/asdf/0.1.0/core/history_entry"
+title: |
+  An entry in the file history.
+
+tag: "tag:stsci.edu:asdf/0.1.0/core/asdf"
+type: object
+properties:
+  description:
+    description: |
+      A description of the transformation performed.
+    type: string
+
+  time:
+    description: |
+      A timestamp for the operation.
+    type: string
+    format: date-time
+
+  software:
+    description: |
+      A description of the software that performed the operation.
```

Comment:
The `asdf_library` member is always the last library that updated the file.  Since -- other than updating binary data in place, I suppose -- it's sort of irrelevant what the previous ASDF library was.

The `history` list is meant to be a history of modifications to the file and to include "higher-level" software, such as calibration pipelines.  That does raise the question of how much software detail to include -- for example, a JWST calibration pipeline will be using a particular version of astropy, numpy, Python, OS etc..  We could include those here (and thus make `software` a list), but I worry that could get pretty out-of-hand.